src/components/app-detail/app-detail.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-card dark>
+    <v-card-text class="d-flex flex-no-wrap justify-space-between">
+      <div>
+        <v-card-title class="headline white--text">Detail</v-card-title>
+      </div>
+    </v-card-text>
+    <v-container>
+      <v-row dense>
+        <v-col cols="3">
+          <v-avatar class="ma-3" size="125" tile>
+            <img :src="detailed.artworkUrl600 || detailed.artworkUrl100" />
```

Comment:
Probably you can use a computed property here.

Reply:
detailed.artworkUrl600 was removed, data would only contain detailed.artworkUrl100 prop

Reply:
👌